docs/components/connectors/custom-built-connectors/connector-sdk.md

```diff
@@ -1087,7 +1088,7 @@ public class Main {
 
     zeebeClient.newWorker()
         .jobType("io.camunda:template:1")
-        .handler(new ConnectorJobHandler(new MyConnectorFunction()))
+        .handler(new ConnectorJobHandler(new MyConnectorFunction(), new DefaultValidationProvider()))
```

Comment:
Looking at the actual code I am wondering whether this constructor is even supported anymore beside testing purposes.

As the execution requires more dependencies to be in place beside the validation provider, see https://github.com/camunda/connectors-bundle/blob/9e427e9cc83f21a8f0b576b37ef36c21faa5fcdc/connector-runtime/connector-runtime-core/src/main/java/io/camunda/connector/runtime/core/outbound/ConnectorJobHandler.java#L188

cc @chillleader wdty?